chains/geth/chain.go

```diff
@@ -21,10 +21,10 @@ import (
 
 type Chain struct {
 	chainID        int64
```

Comment:
btw, you can get a chain-id via `eth_chainId` method

Reply:
Sorry, what does this mean?
Where is the method eth_chainId?

Reply:
sorry, `eth_chainId ` is a method of json rpc provided by ethereum client

Reply:
https://eips.ethereum.org/EIPS/eip-695

Reply:
Ah, so that's how it is. I understand

Reply:
https://github.com/datachainlab/ethereum-ics20-cli/pull/3/commits/febc444353171ca1910dec94a9d8c299ab4165a8

use `eth_chainId` method and removed chainId flag from commands